views/partials/hipb_attribution.hbs

```diff
@@ -0,0 +1 @@
+ <span class="source-info">Breach data provided by<a href="https://www.haveibeenpwned.com" target="_blank">Have I Been Pwned</a></span>
```

Comment:
we should add the `rel="noopener noreferrer"` to the `<a target="_blank">` tag to avoid leaking referrer information.

Plus, I think we're missing a space between the "by" and the `<a>` tag.